production/helm/loki/templates/backend/service-backend.yaml

```diff
@@ -8,9 +8,11 @@ metadata:
   namespace: {{ $.Release.Namespace }}
   labels:
     {{- include "loki.backendLabels" . | nindent 4 }}
-    {{- with .Values.backend.serviceLabels }}
-    {{- toYaml . | nindent 4 }}
-    {{- end }}
+    {{- toYaml .Values.loki.serviceLabels | nindent 4}}
```

Comment:
I think these may be failing to render when empty. Wrapping them in `with` statements should help

Reply:
Can you give an example when you think this fails? I tested it and I don't see it causing issues. Without the `with` statement it's just a lot shorter.

Reply:
anyhow, I have changed it to use with, doesn't really matter too much to me.